UnityProject/Assets/Scripts/Player/PlayerParticle.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using Logs;
+using Mirror;
+using UnityEngine;
+
+namespace Player
+{
+	public class PlayerParticle : NetworkBehaviour
+	{
+		[SerializeField] private Transform particleHolder;
+		private List<PlayerParticleObject> particles = new List<PlayerParticleObject>();
+
+		private void Start()
+		{
+			for (int i = 0; i < particleHolder.childCount - 1; i++)
+			{
+				var obj = particleHolder.GetChild(i).gameObject;
+				PlayerParticleObject particleObject = new PlayerParticleObject()
+				{
+					Reference = obj,
+				};
+				particles.Add(particleObject);
+			}
+		}
+
+		[ClientRpc]
+		public void ToggleParticle(string id, bool state)
```

Comment:
this should be a syncvar? because it's toggling so therefore it has a trackable state?

Reply:
done